src/VisualStudio/Core/Test/ChangeSignature/AddParameterViewModelTests.vb

```diff
@@ -305,7 +306,7 @@ class MyClass
                     Assert.True(False, "Missing caret location in document.")
                 End If
 
-                Dim viewModel = New AddParameterDialogViewModel(workspaceDoc, doc.CursorPosition.Value)
+                Dim viewModel = New AddParameterDialogViewModel(workspaceDoc, doc.CursorPosition.Value, New TestNotificationService())
```

Comment:
❗ `TestNotificationService` is a MEF part, and must not be directly constructed. If you want to use this instance, place it in the MEF catalog prior to the creation of the export provider used for the test workspace.